Content.Shared/Traits/Assorted/LightweightComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Shared.Traits.Assorted;
+
+/// <summary>
+/// Used for the lightweight trait.
+/// </summary>
+[RegisterComponent]
```

Comment:
This comment should better explain that it's related to drunksystem and what it does, it also needs networking.